### DISCUSSION TITLE
postal initialize wont work anymore

### DISCUSSION THREAD

**Discussion by Itseric100** [postalserver/postal#2436] [Help with using Postal]
I see there is a new version that was just released and i am wondering how this is affecting the old version, i tried to install postal and when i input postal initialize, it says postal already exists. It allows with the proceed but on trying to send an email, it wouldnt send

**Comment by willpower232** [comment]
`postal initialize` is only for installing Postal the first time, you should not run that again.

There is a built in `postal upgrade` which does most of the work for you, have a look at the documentation https://docs.postalserver.io/install/upgrading
